src/DynamoCoreWpf/Views/GuidedTour/PopupWindow.xaml.cs

```diff
@@ -23,10 +23,13 @@ public PopupWindow(PopupWindowViewModel viewModel, HostControlInfo hInfo)
 
             //Due that we are drawing the Direction pointers on left and right side of the Canvas (10 width each one) then we need to add 20
             RootLayout.Width = popupViewModel.Width + 20; 
-            RootLayout.Height = popupViewModel.Height;
+            //Also a shadow of 10 pixels in top and 10 pixels at the bottom will be shown then we need to add 20
+            RootLayout.Height = popupViewModel.Height + 20;
```

Comment:
Let's define this 10 in the  class and use 10 * 2 here, maybe there will be clearer

Reply:
@QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/12048/commits/c84d82b2d65d9ce4ea1dcf90649974be6f4618cd